ramls/examples/outboxEventLog.sample

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Where is this example used?

As I understand it, these JSON schema are only defined in order to trigger code generation, and thus are not defined in any endpoint definitions, which is where examples would typically be used.

Reply:
You're right. Removed the sample.